docs/block-client-apis/checkout/checkout-api.md

```diff
@@ -47,13 +113,13 @@ The customer data context exposes the api interfaces for the following things vi
 
 The shipping method data context exposes the api interfaces for the following things (typedef `ShippingMethodDataContext`) via the `useShippingMethodData` hook:
 
-- `shippingErrorStatus`: The current error status for the context.
-- `dispatchErrorStatus`: A function for dispatching a shipping error status. Used in combination with...
-- `shippingErrorTypes`: An object with the various error statuses that can be dispatched (`NONE`, `INVALID_ADDRESS`, `UNKNOWN`)
-- `onShippingRateSuccess`: This is a function for registering a callback to be invoked when shipping rates are retrieved successfully. Callbacks will receive the new rates as an argument.
-- `onShippingRateFail`: This is a function for registering a callback to be invoked when shipping rates fail to be retrieved. Callbacks will receive the error status as an argument.
-- `onShippingRateSelectSuccess`: This is a function for registering a callback to be invoked when shipping rate selection is successful.
-- `onShippingRateSelectFail`: This is a function for registering a callback to be invoked when shipping rates selection is unsuccessful.
+-   `shippingErrorStatus`: The current error status for the context.
+-   `dispatchErrorStatus`: A function for dispatching a shipping error status. Used in combination with...
+-   `shippingErrorTypes`: An object with the various error statuses that can be dispatched (`NONE`, `INVALID_ADDRESS`, `UNKNOWN`)
+-   `onShippingRateSuccess`: This is a function for registering a callback to be invoked when shipping rates are retrieved successfully. Callbacks will receive the new rates as an argument.
+-   `onShippingRateFail`: This is a function for registering a callback to be invoked when shipping rates fail to be retrieved. Callbacks will receive the error status as an argument.
+-   `onShippingRateSelectSuccess`: This is a function for registering a callback to be invoked when shipping rate selection is successful.
+-   `onShippingRateSelectFail`: This is a function for registering a callback to be invoked when shipping rates selection is unsuccessful.
```

Comment:
Could we discard these changes if all they do is formatting?

Reply:
It's just prettier doing its thing. I think it's ok no?

Reply:
Ok then 😁 I didn't know if it was your IDE, but if it's a repo-wide rule then great, thanks!